static.config.js

```diff
@@ -33,12 +34,16 @@ export default {
     const { allServicePages } = await enGraphQLClient.request(allServicePagesQuery)
     const { allTopics } = await enGraphQLClient.request(allTopicPagesQuery)
     const { allDepartments } = await enGraphQLClient.request(allDepartmentPagesQuery)
+    const { allServicePages: topServices } = await enGraphQLClient.request(topServicesQuery)
```

Comment:
super minor, but could be update the allServicePages var naming everywhere related to the homepage to be more descriptive to the returned data if we decide to keep this query. I got confused about why we were pulling all service page data on the homepage.

Reply:
another good call.